ACE/ace/Module.h

```diff
@@ -59,6 +59,8 @@ class ACE_Export ACE_Module_Base
     /// Indicates that close() should not delete any Tasks.
     M_DELETE_NONE = 4
   };
+
+  inline virtual ~ACE_Module_Base (void) {}
```

Comment:
ACE-guidelines.html says this shouldn't be inline ("If a class has any virtual functions..."). Not sure it's still a concern, but it does make sense that for efficient code generation / linking there should be a single object file for the vtable/typeinfo.

Reply:
Yes, you are right

Reply:
that didn't work (on Win32) for some reason, removed this change for now